lib/Conversion/PolynomialToStandard/PolynomialToStandard.cpp

```diff
@@ -439,7 +444,10 @@ struct ConvertMul : public OpConversionPattern<MulOp> {
     // set the dense elements attr properly based on that (i.e., what if it's an
     // i16 or i128?)
     auto polymulOutput = b.create<arith::ConstantOp>(
-        polymulTensorType, DenseElementsAttr::get(polymulTensorType, 0L));
+        polymulTensorType,
+        DenseElementsAttr::get(
+            polymulTensorType,
+            APInt(polymulTensorType.getElementTypeBitWidth(), 0L)));
```

Comment:
does this fix the TODO comment above since you're using the APInt with the element bit width now?

Reply:
Done.